react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+export default class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { indexOfDisplayed: 0 };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    clearInterval(this.ID);
+    let newIndex;
+    if (event.target.getAttribute('id') === '5') {
+      newIndex = this.state.indexOfDisplayed > 0 ? this.state.indexOfDisplayed - 1 : this.props.imageList.length - 1;
+    } else if (event.target.getAttribute('id') === '6') {
+      newIndex = this.state.indexOfDisplayed < this.props.imageList.length - 1 ? this.state.indexOfDisplayed + 1 : 0;
+    } else {
+      newIndex = parseInt(event.target.getAttribute('id'));
+    }
```

Comment:
I'm not really clear on what the significance of `'5'` and `'6'` are. What does it mean if these are clicked?